### ISSUE TITLE
SVGFEDropShadowElement missing in dom typings

### ISSUE THREAD

**Issue by Fuzzyma** [microsoft/TypeScript#29966]
**TypeScript Version:**  Looked into the current master - its not there

**Search Terms:** SVGFEDropShadowElement

**Expected behavior:** SVGFEDropShadowElement is defined

**Actual behavior:** SVGFEDropShadowElement is not defined

MDN docs for the element: https://developer.mozilla.org/en-US/docs/Web/API/SVGFEDropShadowElement

**Comment by Fabianopb** [comment]
I'd be happy to contribute to this one!

**Comment by Fuzzyma** [comment]
@Fabianopb should be something around the lines

```
  interface SVGFEDropShadowElement extends SVGElement, SVGFilterPrimitiveStandardAttributes {
    readonly in1: SVGAnimatedString;
    readonly dx: SVGAnimatedNumber;
    readonly dy: SVGAnimatedNumber;
    readonly stdDeviationX: SVGAnimatedNumber;
    readonly stdDeviationY: SVGAnimatedNumber;
    setStdDeviation(stdDeviationX: number, stdDeviationY: number): void;
    addEventListener<K extends keyof SVGElementEventMap>(type: K, listener: (this: SVGFEDisplacementMapElement, ev: SVGElementEventMap[K]) => any, options?: boolean | AddEventListenerOptions): void;
    addEventListener(type: string, listener: EventListenerOrEventListenerObject, options?: boolean | AddEventListenerOptions): void;
    removeEventListener<K extends keyof SVGElementEventMap>(type: K, listener: (this: SVGFEDisplacementMapElement, ev: SVGElementEventMap[K]) => any, options?: boolean | EventListenerOptions): void;
    removeEventListener(type: string, listener: EventListenerOrEventListenerObject, options?: boolean | EventListenerOptions): void;
  }
```

**Comment by Fabianopb** [comment]
Thanks for the tip @Fuzzyma, PR created in TSJS-lib-generator: https://github.com/Microsoft/TSJS-lib-generator/pull/667

**Comment by IllusionMH** [comment]
Looks like this one can be closed.
PR was merged and these types are available since TS 3.5 at least
